lang/pl.json

```diff
@@ -247,7 +247,28 @@
 			"rollOption": "Opcja rzutu, która aktywuje ten zestaw animacji (np. <code>item:slug:jaws</code> lub <code>check:statistic:society</code>).<p />Wybór powinien być dostosowany do animowanej rzeczy. Jeśli masz wątpliwości, użyj opcji <b>Historia animacji</b> aby znaleźć odpowiednią opcję rzutu dla danej akcji.",
 			"triggers": "Lista wyzwalaczy, które spowodują wykonanie tej animacji.",
 			"overrides": "Lista opcji rzutów, z których każda odpowiada głównemu predykatowi animacji. Jeśli dwie animacje są wyzwalane przez ten sam wyzwalacz i jedna z nich zastępuje drugą, nadrzędna animacja nie zostanie wykonana.",
-			"predicates": "Lista predykatów zgodnie z systemem pf2e. Animacja zostanie wykonana tylko wtedy, gdy te warunki mają zastosowanie."
+			"predicates": "Lista predykatów zgodnie z systemem pf2e. Animacja zostanie wykonana tylko wtedy, gdy te warunki mają zastosowanie.",
+			"label": "Rozróżniający wielkość liter ciąg znaków, który istnieje, aby inne jednostki mogły się do niego odwoływać, szczególnie gdy inny pakiet „usuwa” go. Powinien on być w miarę unikalny, aby uniknąć kolizji.",
```

Comment:
To jest bardzo zły początek zdania wyjaśnia prostą idee "Tytułu." W takich sytuacjach proszę przenieść tego typu zasady gdzieś indziej.